src/languages/ko-KR.js

```diff
@@ -361,8 +361,8 @@ export default {
     topicExplanation: "모니터링할 MQTT Topic",
     successMessage: "성공 메시지",
     successMessageExplanation: "성공으로 간주되는 MQTT 메시지",
-    error: "error",
-    critical: "critical",
+    error: "오류",
+    critical: "크리티컬",
```

Comment:
```suggestion
    critical: "치명적인",
```
I guess this wording is better. How about you?

Reply:
I don't think it's good because it's too translated.